src/EditorFeatures/CSharpTest/Diagnostics/Suppression/RemoveUnnecessaryPragmaSuppressionsTests.cs

```diff
@@ -111,7 +111,8 @@ public override void Initialize(AnalysisContext context)
         protected sealed class CompilationEndDiagnosticAnalyzer : DiagnosticAnalyzer
```

Comment:
❔ Do we need any tests for compilation end analyzers that _don't_ have the tag?

Reply:
We do - there was a test failure with just the product changes, which had to be fixed with https://github.com/dotnet/roslyn/pull/48762/commits/e9ee72962adf689093fe9ec87e13bf0be32c6e7c

Reply:
Right, by adding the tag it looks like we no longer have tests for the case _without_ the tag. I would have expected the fix in e9ee729 to instead be implemented by:

1. Creating a copy of the current `CompilationEndDiagnosticAnalyzer` and adding the tag there
2. Adding a second test alongside the one that failed, where the new test references the new analyzer from (1) and the old test has its assertions updated to account for the fact that the old analyzer doesn't have the new tag